corehq/warehouse/transforms/sql/app_status_fact.sql

```diff
@@ -0,0 +1,45 @@
+SELECT * FROM
```

Comment:
yep agreed. i wanted to leave this here because it's useful for me to think about the query / tables we'll need in order to get the desired app status report